copy-alerts/copy-alerts.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Prompt user for namespace and validate
+read -p "Enter the namespace for your kubecost deployment: " NAMESPACE
```

Comment:
i would instead accept this as an arg. makes it easier to script

Reply:
Fixed it.